integration-tests/kyuubi-jdbc-it/pom.xml

```
@@ -133,6 +133,13 @@
                                     <overWrite>true</overWrite>
                                     <outputDirectory>${project.build.directory}</outputDirectory>
                                 </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.apache.kyuubi</groupId>
+                                    <artifactId>kyuubi-hive-jdbc-shaded</artifactId>
```

let's make it respect `-Pjdbc-shaded`
```suggestion
                                    <artifactId>${hive.jdbc.artifact}</artifactId>
```

---

I see, the existing approach does not handle transitive deps properly, leave it then